src/containers/ResetPassword/ResetPassword.js

```diff
@@ -0,0 +1,146 @@
+import React from "react";
+import { Form, Field } from "react-final-form";
+import Avatar from "@material-ui/core/Avatar";
+import Button from "@material-ui/core/Button";
+import CssBaseline from "@material-ui/core/CssBaseline";
+import Box from "@material-ui/core/Box";
+import LockOutlinedIcon from "@material-ui/icons/LockOutlined";
+import Typography from "@material-ui/core/Typography";
+import Container from "@material-ui/core/Container";
+import { makeStyles } from "@material-ui/core/styles";
+
+function ResetPassword() {
+  const classes = useStyles();
+
+  // it should return new password to backend
+  // it will be updated later on!
+  const onSubmit = () => {
+    return null;
+  };
+
+  return (
+    <Container component="main" maxWidth="sm">
+      <CssBaseline />
+      <div className={classes.resetPassword}>
+        <Avatar className={classes.avatar}>
+          <LockOutlinedIcon />
+        </Avatar>
+        <Typography component="h1" variant="h5">
+          Reset Password
+        </Typography>
+        <Form
+          onSubmit={onSubmit}
+          validate={(values) => {
```

Comment:
I think it will be more readable if you separated the logic to a function like onSubmit